### PR TITLE
converting dependency package name format

### PR DESCRIPTION
**Pull request by psuhailp**
This should fix [https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=840207](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=840207)

### CODE REVIEW

File: npm2deb/__init__.py [hunk 1]
```diff
@@ -457,7 +457,10 @@ def _get_Depends(self):
                     mapper.append_warning('error', dep, 'dependency %s '
                                           'not in debian' % (name))
                 version = dependencies[dep].replace('~', '')
+                version = dependencies[dep].replace('^', '')
                 if version:
+                    if version.find('.x') is not -1:
```
Comment (shanavas786) [inline npm2deb/__init__.py]: Is this line necessary ?

Comment (shanavas786) [inline npm2deb/__init__.py]: I meant `if version.find('.x') is not -1` line,
Isn't `version = version.replace('.x', '.0')` sufficient without if block ?

Comment (psuhailp) [inline npm2deb/__init__.py]: yea, that makes sense. Removed that line. Thanks.

**Comment by LeoIannacone** [comment]
LGTM :+1:

File: npm2deb/__init__.py [hunk 2]
```diff
@@ -457,7 +457,9 @@ def _get_Depends(self):
                     mapper.append_warning('error', dep, 'dependency %s '
                                           'not in debian' % (name))
                 version = dependencies[dep].replace('~', '')
+                version = dependencies[dep].replace('^', '')
                 if version:
+                    version = version.lower().replace('.x', '.0')
```
Comment (LeoIannacone) [inline npm2deb/__init__.py]: Maybe we can have something like:
```
version = dependencies[dep]
if version:
    version = version.lower().replace('~', '').replace('^', '').replace('.x', '.0')
```

Comment (psuhailp) [inline npm2deb/__init__.py]: Yeah. That's better. I have changed them.

**Comment by LeoIannacone** [comment]
Thanks!

**Pull request merged**